x-pack/test/cloud_security_posture_functional/cloud_tests/agentless_api_sanity.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+// import { CLOUD_CREDENTIALS_PACKAGE_VERSION } from '@kbn/cloud-security-posture-plugin/common/constants';
+import type { FtrProviderContext } from '../ftr_provider_context';
+// eslint-disable-next-line import/no-default-export
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const queryBar = getService('queryBar');
+  const pageObjects = getPageObjects(['common', 'header', 'cisAddIntegration', 'findings']);
+
+  // let cisIntegration: typeof pageObjects.cisAddIntegration;
```

Comment:
The commented code in this file will be enabled after
-  https://github.com/elastic/kibana/issues/174598